internal/extsvc/codehost.go

```diff
@@ -24,10 +24,14 @@ var (
 	MavenURL    = &url.URL{Host: "maven"}
 	JVMPackages = NewCodeHost(MavenURL, TypeJVMPackages)
 
+	NpmURL      = &url.URL{Host: "npm"}
+	NPMPackages = NewCodeHost(NpmURL, TypeNPMPackages)
```

Comment:
```suggestion
	NPMURL      = &url.URL{Host: "npm"}
	NPMPackages = NewCodeHost(NPMURL, TypeNPMPackages)
```
(For consistency with other usages.)